packages/back-end/src/integrations/Mssql.ts

```diff
@@ -28,6 +28,7 @@ export default class Mssql extends SqlIntegration {
       user: this.params.user,
       password: this.params.password,
       database: this.params.database,
+      requestTimeout: (this.params.requestTimeout ?? 0) * 1000,
```

Comment:
Do you also need to set `connectionTimeout` to the same value, or let customers customize both?

Reply:
I thought about this—I have no problem adding, but it seems reasonable for the client to connect to the database within 15 seconds but unreasonable for all queries to complete within 15 seconds. I have no problem adding it if you think it's important.